chain/network/src/routing/routing.rs

```diff
@@ -85,6 +85,22 @@ impl Edge {
         Edge(Arc::new(EdgeInner::new(peer0, peer1, nonce, signature0, signature1)))
     }
 
+    pub fn nonce(&self) -> u64 {
```

Comment:
@matklad - is it the right way to do "getter" accessors ? or does Rust offer some other option ?

Reply:
@mm-near Yes, @mm-near said in another PR that's how you do it in Rust.